src/cli/options.js

```diff
@@ -70,8 +74,8 @@ module.exports = optionator({
     },
     {
       option: 'version',
-      alias: 'v',
       type: 'Boolean',
+      alias: 'r',
```

Comment:
I'd rather keep 'v' for version, lets discuss this in channel!

Reply:
I agree, put `vb` to `verbose` and change `r` to `v` for version

Reply:
I founded a better shorthand for `verbose`, keep `v` for `version` and use `V` for `verbose`